src/lib/input/input-container.spec.ts

```diff
@@ -775,6 +932,41 @@ class MdTextareaWithBindings {
 })
 class MdInputContainerMissingMdInputTestController {}
 
+@Component({
+  template: `
+    <form #form="ngForm" novalidate>
+      <md-input-container>
+        <input mdInput [formControl]="formControl">
+        <md-hint>Please type something</md-hint>
+        <md-error *ngIf="renderError">This field is required</md-error>
+      </md-input-container>
+    </form>
+  `
+})
+class MdInputContainerWithFormErrorMessages {
+  @ViewChild('form') form: NgForm;
+  formControl = new FormControl('', Validators.required);
+  renderError = true;
+}
+
+@Component({
+  template: `
+    <form [formGroup]="formGroup" (ngSubmit)="onSubmit()" novalidate>
+      <md-input-container>
+        <input mdInput [formControl]="formControl">
```

Comment:
I think you mean `formControlName` here? `formControl` is for individual form controls, not controls within groups.